src/common/tlv.hpp

```diff
@@ -171,6 +174,24 @@ class Tlv
      */
     Tlv *GetNext(void) { return reinterpret_cast<Tlv *>(static_cast<uint8_t *>(GetValue()) + GetLength()); }
 
+    void SetUdpSourcePort(uint16_t srcPort)
+    {
+        uint16_t *value = reinterpret_cast<uint16_t *>(GetValue());
```

Comment:
Is `GetValue()` guaranteed to be aligned to `uint16_t`? How about encoding in network byte order?

Reply:
Function for udp proxy, not used in this pull request. Code removed.